### DISCUSSION TITLE
유니티 협업프로젝트 커밋 관련 질문!

### DISCUSSION THREAD

**Discussion by fkdl0048** [fkdl0048/Mentoring-Project#2] [Q&A] [answered]
어제 멘토링에서 풀리퀘스트로 관리하는 방법을 추천해주셨는데 현재 CSV파싱 관련 기능을 추가하여 프로젝트에 올릴려고 할때 
따로 브랜치를 파서 풀리퀘스트로 올리고 점검 후에 merge하는 방식이 좋은 방법 맞나요?

- 브랜치 이름은 해당 기능이여야 하는지 or 작성자 이름이여야 하는지

**Comment by jongfeel** [comment, marked as answer]
그건 협의하기 나름일 것 같습니다.

보통은 pull request 올릴 때 작성자가 누군지는 아니까 기능 단위로 branch 이름을 만들기는 합니다.
그런데 branch naming rule은 정해진 것이 없으니까 `작성자:기능` 으로 써도 좋을 수 있어요.

그리고 리뷰 후에 merge가 들어가는 방식이 절차일 수도 있지만 서로 약속한 규칙대로 소스 코드가 들어가는 거라 이해하면
결과적으로는 좋은 방식이 맞긴 합니다.
히스토리 추적 할 때도 좋고요.

**Reply by fkdl0048** [reply]
말씀하신 대로 한번 작업해봤는데 나중에 시간 괜찮으시면 확인 부탁드리겠습니다..!

https://github.com/Bridge-Project-2022/Bridge_Project_git

로컬에서 작업하고 브랜치 만들고 해당 브랜치 푸쉬 후 풀리퀘 작업 진행하였습니다. 브랜치는 삭제했습니다..! 
(작업은 csv파싱관련 작업을 했는데 혹시 코드 리뷰는 10/17날 여유가 된다면? 받을 수 있을까요?)

**Reply by jongfeel** [reply]
아 그러면 리뷰할 목록으로 남겨 두면 좋을 것 같습니다.
